server/asset/eth/eth.go

```diff
@@ -65,11 +65,11 @@ func (d *Driver) Setup(configPath string, logger dex.Logger, network dex.Network
 
 // DecodeCoinID creates a human-readable representation of a coin ID for Ethereum.
 func (d *Driver) DecodeCoinID(coinID []byte) (string, error) {
-	coinId, err := DecodeCoinID(coinID)
```

Comment:
What's your vision for removing the exported `DecodeCoinID` that switches on a `CoinIDFlag`?

Reply:
`DecodeCoinID` is moved in #1301. To **dex/networks/eth/cointypes.go ** for now, but may end up in **client/asset/eth** since I don't think we'll end up using them at all on the server.